src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/AppleCertificatePal.Pem.iOS.cs

```diff
@@ -2,59 +2,48 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Buffers;
+using System.Buffers.Text;
 using System.Diagnostics;
-using System.Text;
-using Internal.Cryptography;
 
 namespace System.Security.Cryptography.X509Certificates
 {
     internal sealed partial class AppleCertificatePal : ICertificatePal
     {
-        internal delegate bool DerCallback(ReadOnlySpan<byte> derData, X509ContentType contentType);
```

Comment:
We don't need a dedicated delegate anymore with `allows ref struct` on Func now, so use Func.